From e7a1539cc81be4fe91e0dc88b7842a5a76301bf4 Mon Sep 17 00:00:00 2001
From: Meng Mingming <mingming.meng@nxp.com>
Date: Wed, 21 Dec 2016 17:01:21 +0800
Subject: [PATCH 4377/5242] MGS-2487 gpu: Set gpu physical base address and
 size into dts for 7ulp

commit  7c801a3bb512ec7a0280e49c2d7355e4c33b622e from
https://source.codeaurora.org/external/imx/linux-imx.git

Set gpu physical base address and size into dts for 7ulp.

Date: Dec 21, 2016
Signed-off-by: Meng Mingming <mingming.meng@nxp.com>
(cherry picked from commit b5ae6ee5c84b8eb197af34d78f572be667ed0574)
(cherry picked from commit 64ba25ed48ec760a5b783ecf3c0defb5d71916be)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx7ulp.dtsi b/arch/arm/boot/dts/imx7ulp.dtsi
index 3f341bf..b173508 100644
--- a/arch/arm/boot/dts/imx7ulp.dtsi
+++ b/arch/arm/boot/dts/imx7ulp.dtsi
@@ -656,7 +656,7 @@
 		gpu: gpu@41800000 {
 			compatible = "fsl,imx6q-gpu";
 			reg = <0x41800000 0x80000>, <0x41880000 0x80000>,
-				<0x0 0x0>, <0x0 0x4000000>;
+				<0x60000000 0x40000000>, <0x0 0x4000000>;
 			reg-names = "iobase_3d", "iobase_2d",
 				"phys_baseaddr", "contiguous_mem";
 			interrupts = <GIC_SPI 44 IRQ_TYPE_LEVEL_HIGH>,
-- 
1.7.9.5

